nixos/modules/virtualisation/xen-dom0.nix

```diff
@@ -224,229 +918,158 @@ in
         (isYes "XEN_BALLOON")
         (isYes "XEN_SCRUB_PAGES")
       ];
+    };
 
+    # Enable the xen:/// connection in virt-manager by default if the user has the module enabled.
+    programs.dconf.profiles.user.databases = lib.mkDefault lib.lists.optional config.programs.virt-manager.enable {
+      settings."org/virt-manager/virt-manager/connections" = {
+        autoconnect = [ "xen:///system" ];
+        uris = [ "xen:///system" ];
+      };
+    };
```

Comment:
I’m iffy about writing to user dconf configuration in a module like this. I don’t think anything else does this and it should probably at least be a separate opt‐in thing.

Reply:
I'll just stash the libvirt/virt-manager stuff and work on something more appropriate later. Interested users can set this up themselves anyway.

Reply:
virt-manager itself does that for qemu:
https://github.com/NixOS/nixpkgs/blob/nixos-24.05/nixos/modules/programs/virt-manager.nix#L15-L26

So having that continues the trend "virt-manager lists all the available connections by default"

Reply:
I think it's a better idea to merge these configurations by editing the virt-manager module. It'll also help whoever is maintaining virt-manager to know what modules are using virt-manager.